docs/csharp/programming-guide/xmldoc/toc.yml

```diff
@@ -3,6 +3,13 @@
   items:
```

Comment:
I just noticed this folder is missing the index page so the URL doesn't work if we leave just https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/

The first node also needs to be expanded to become the parent node